src/aspire/utils/blk_diag_matrix.py

```diff
@@ -99,9 +100,9 @@ def eye(blk_partition, dtype=np.float64):
 
         :param blk_partition: The matrix block partition in the form of a
             K-element list storing all shapes of K diagonal matrix blocks,
-            where `blk_partition[i]` corresponds to the shape (number of rows and columns)
+            where `blk_partition[i]` corresponds to the shape (num of rows cols)
```

Comment:
"num of rows cols" should be consistent with previous cases.

Reply:
Sure, I considered them obvious abbreviations and it seemed more effective than refactoring pages of doc sting line breaks.  Usually that's not the sort of thing people want me to spend time on, but I'll happily fix it up.  Thanks for suggestions. Have a nice weekend.